modules/update-process-workflow.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-process-workflow_{context}"]
+= Update process workflow
+
+The following represents a detailed workflow of the {product-title} (OCP) update process.
```

Comment:
```suggestion
The following represents a detailed workflow of the {product-title} (OCP) update process:
```

Reply:
s/The following represents/The following steps represent  
Not expressly stated in the ISG. But the examples in the ISG seem to follow this format.